packages/react-aria-components/test/Button.test.js

```diff
@@ -137,4 +138,63 @@ describe('Button', () => {
     fireEvent.mouseUp(button);
     expect(button).toHaveTextContent('Test');
   });
+
+  // isPending state
+  it('displays a spinner when isPending prop is true', async function () {
```

Comment:
Shouldn't we have a test that confirms the button leaves the isPending state correctly? The tests could start with a button as pending and we update the `isPending` prop to change the state, the inverse of this test as far as initial state and onPress behavior?